app/src/main/res/values/strings.xml

```diff
@@ -12,10 +12,18 @@
     <string name="game">Game</string>
     <string name="add">Add</string>
     <string name="update">Update</string>
-    <string name="hours">Hours</string>
+    <string name="hours"> Hours</string>
     <string name="sides">Pages</string>
     <string name="backlog">Backlog</string>
     <string name="inProgress">In Progress</string>
     <string name="finished">Finished</string>
     <string name="cancelled">Cancelled</string>
+    <string name="back">Back</string>
+    <string name="reroll">Reroll</string>
+    <string name="randFragError">"The provided List contains no Elements that haven\'t been watched yet.\n Please check if u have trackables that have the status in Progress!"</string>
```

Comment:
Hier sollte die Info dann auch sagen, dass man trackables in progress oder backlog braucht